aten/src/ATen/core/ivalue.cpp

```diff
@@ -30,9 +30,9 @@ void checkCustomClassType(const Type* expected_type, const Type* actual_type) {
   // Type's, this needs to be changed!
   TORCH_CHECK(actual_type == expected_type,
               "Tried to convert an IValue of type ",
-              actual_type->repr_str(),
+              actual_type ? actual_type->repr_str() : std::string("*NULL*"),
```

Comment:
If those should _not_ be NULL, maybe we can add a couple of additional checks before this:
```
TORCH_CHECK(actual_type);
TORCH_CHECK(expected_type);
TORCH_CHECK(actual_type == expected_type, ...):
```